react-app/src/History.js

```diff
@@ -42,7 +47,7 @@ const styles = StyleSheet.create({
   calendar: {
     // to reviewers
```

Comment:
Are these comments need? If not, I suggest we clean them up

Reply:
yes we can clean up
※I just wanted to explain how  "...StyleSheet.absoluteFillObject" works